internal/datastore/mysql/migrations/zz_migration.0002_namespace_tables.go

```diff
@@ -0,0 +1,54 @@
+package migrations
+
+import (
+	"fmt"
+)
+
+// namespace max size: https://buf.build/authzed/api/file/main/authzed/api/v0/core.proto#L29
```

Comment:
Can we read them from the code?

Reply:
Are you suggesting to change the DML statements to interpolate the size from a constant? Such constants do not exist anywhere in the codebase, the numbers are hardcoded. Or are you suggesting that I should create a new constant in this file and interpolate the DML?

Reply:
> Can we read them from the code?

I would like that but not that I think due to the way the protobuf generation works the validation is handled inline: https://github.com/authzed/spicedb/blob/main/pkg/proto/core/v1/core.pb.validate.go#L240-L249